spec/requests/v1/submissions_spec.rb

```diff
@@ -156,6 +159,33 @@
         expect(response).to have_http_status(:no_content)
       end
     end
+
+    context 'setting total_value based on the framework definition' do
```

Comment:
This doesn't feel like it's really a separate context. i.e. There is nothing different about the "context" here compared to the other "POST creates an entry" ones. Because we expect the total_value to be set every time an entry is created I think we should simply put the expectation into all the existing contexts where we'd expect it to happen.